js/runner_test.go

```diff
@@ -178,8 +178,9 @@ func TestOptionsSettingToScript(t *testing.T) {
 func TestOptionsPropagationToScript(t *testing.T) {
 	t.Parallel()
 	data := `
-			export let options = { setupTimeout: "1s", myOption: "test" };
-			export default function() {
+			var options = { setupTimeout: "1s", myOption: "test" };
+			exports.options = options;
```

Comment:
I once knew why this is required ... but not anymore I will need to go figure it out again :( 
It has something to do with the way we bind `exports`, but I decided to figure it out later

Reply:
okay so if you just do:
`export.options = {}` you don't have any local variable `options`, but you should be able to just use `exports.options` but there was a bug https://github.com/loadimpact/k6/pull/1430 .

 I can rebase only the fix on top of new-schedulers/master ? if we aren't going to merge this soon